en/docs/fragments/subject-attribute-oidc.md

```diff
@@ -0,0 +1,19 @@
+Choose an attribute from the local attributes listed in the **Subject attribute** dropdown. 
+
+!!! note 
+    If you want to choose an attribute from the local claim dialect from the dropdown that is not present in `http://wso2.org/oidc/claim`, you need to first add it  to the OpenID Connect dialect. Follow the instructions given below to do this. 
```

Comment:
from the local claim dialect from the dropdown --> from the local claim dialect dropdown list

Reply:
you need to first add it --> first add it (you need to is unnecessary there)

Reply:
there is an extra space after the word "it"